include/eve/function/factorial.hpp

```diff
@@ -38,7 +38,7 @@ namespace eve
   //!
   //! **Return value**
   //!
-  //! The result element type is always double to try to avoid overflow
+  //! The result [element type](eve::element_type_t) is always double to try to avoid overflow
```

Comment:
`eve::element_type`, `element_type_t` is a subpart of the documentation of the former

Reply:
done